### DISCUSSION TITLE
What does "cancelled" mean?

### DISCUSSION THREAD

**Discussion by julix-unity** [community/community#14292] [Copilot Conversations] [answered]
![Screen Shot 2022-04-04 at 3 16 21 PM](https://user-images.githubusercontent.com/69219981/161641047-641d4bb7-5030-4d24-816a-c14cfccc4b6d.png)

**Comment by davecheney** [comment, marked as answer]
This is usually an indication of a networking problem. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?
